src/main/java/ch/ivyteam/ivy/maven/AbstractEngineMojo.java

```diff
@@ -41,6 +44,9 @@
   
   protected static final String ENGINE_DIRECTORY_PROPERTY = "ivy.engine.directory";
   
+  @Parameter(property = "project", required = true, readonly = true)
+  protected MavenProject project;
```

Comment:
does the plain 'install-engine-mojo' > comand line usage still work ? I guess not ... project requires mvn to be executed with a project . 
But install-engine-mojo can provide one for other build pipelines than maven. think of a chef or Puppet script to setup your complete axon.ivy environment from scratch....

Reply:
I ran:
mvn com.axonivy.ivy.ci:project-build-plugin:8.0.4-SNAPSHOT:installEngine

without a project and it worked without any issues.